go.mod

```diff
@@ -33,6 +33,11 @@ require (
 	gopkg.in/square/go-jose.v2 v2.6.0
 )
 
+require (
+	cloud.google.com/go/storage v1.10.0
```

Comment:
Were there concerns about pulling in this library? Was it just due to size?

Reply:
Yeah I'd like to replace it with regular HTTP calls instead of taking a dependency on this GCS package. It's not _huge_, it's just mostly unnecessary.